doc/build_apps/auth/jwt.rst

```diff
@@ -180,3 +180,11 @@ Any attempt to add a certificate with mismatching claims in a ``set_jwt_public_s
 
 Some IdPs, like MAA, advertise a mix of SGX and non-SGX signing certificates.
 In this case, ``key_filter`` must be set to ``sgx`` such that only those certificates are stored which contain SGX evidence.
+
+Extracting JWT metrics
+----------------------
+
+CCF tracks the number of JWT queries that were attempted and succeeded. This can be used to identify errors, for example when the number of attempts doesn't match the number of successes.
+Each attempt, for each key on each issuer, is tracked as an attempt, and eventually a success, if the update completes.
```

Comment:
This is per-issuer only, not per-key.

Reply:
Ah, correct! I interpreted the foreach loop the wrong way round, thanks!